src/metaswitch/clearwater/config_manager/etcd_synchronizer.py

```diff
@@ -62,8 +65,13 @@ def main(self):
                 break
 
             if value:
+                _log.info("Got new config value from etcd - filename {}, file size {}, MD5 hash {}".format(
+                    self._plugin.file(),
+                    len(value),
+                    md5(value).hexdigest()))
                 _log.debug("Got new config value from etcd:\n{}".format(value))
```

Comment:
This came out of discussions with @bossmc and @AlexHockey - it's useful to have some way to track how a file has been changed by config manager, but that's developer-level and not appropriate for ENT logs, and the existing log is at DEBUG level so not logged by default. Logging file size and MD5 hash makes it possible to see how a file has changed, in a way that's concise enough to be loggable at INFO level.